examples/dwarfdump.rs

```diff
@@ -1093,8 +1129,9 @@ fn dump_attr_value<R: Reader, W: Write>(
             let address = dwarf.address(unit, index)?;
             writeln!(w, "0x{:08x}", address)?;
         }
-        gimli::AttributeValue::UnitRef(gimli::UnitOffset(offset)) => {
-            writeln!(w, "<0x{:08x}>", offset)?;
+        gimli::AttributeValue::UnitRef(o) => {
+            write_offset(w, unit, o, flags)?;
```

Comment:
I prefer that within attributes we use a format consistent with the [`AttributeValue::DebugInfoRef`](https://github.com/gimli-rs/gimli/blob/423431f6747167af8e64cc8c8378f9a34a18ff8f/examples/dwarfdump.rs#L1100) variant, rather than the `GOFF` label.

Reply:
Also, I think it would be useful to always display both here.